dc.dev.yml

```diff
@@ -11,6 +11,9 @@ services:
     ports:
 #      - '8443:8443'
       - '8080:8080'
+  tomcat_expedius:
+    ports: 
+      - '8081:8081'
```

Comment:
What does exposing this do? Does it have a web UI , or API?

Reply:
Yes. Expedius has a configuration UI that I could not get to work over HTTPS.  Ideally it's: https://<url:port>/Expedius